include/mbedtls/ssl.h

```diff
@@ -1447,6 +1455,12 @@ struct mbedtls_ssl_config {
 #if defined(MBEDTLS_SSL_SESSION_TICKETS) && \
     defined(MBEDTLS_SSL_CLI_C)
     uint8_t MBEDTLS_PRIVATE(session_tickets);   /*!< use session tickets? */
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3)
+    /** Whether we handle NewSessionTicket TLS 1.3 messages (<>0) or just ignore them (==0)
+     *  They are ignored by default.
+     */
+    uint8_t MBEDTLS_PRIVATE(new_session_tickets_enabled);
```

Comment:
BLOCKER: We cannot change the size or layout of `mbedtls_ssl_config` in an LTS branch: it's an ABI break. For example, `mbedtls_ssl_config` might be on the application stack.

We can piggyback on `session_tickets` which currently only uses two values in an 8-bit field.